matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/poll/LoadedPollsStatus.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.room.poll
+
+/**
+ * Represent the status of the loaded polls for a room.
+ */
+data class LoadedPollsStatus(
+        val canLoadMore: Boolean,
+        val nbSyncedDays: Int,
```

Comment:
The `nb` prefix is a bit cryptic. Maybe `numSyncedDays` or `daysSynced` would be clearer?

Reply:
I renamed the field and also added more comments to explain the fields and avoid ambiguity. Let me know what you think of it. See [c7d3e19](https://github.com/vector-im/element-android/pull/8003/commits/c7d3e1926f71c79f127a0fecb0fe39229343bded)

Reply:
Looks great, thank you for the comments!